### ISSUE TITLE
[Question] two ? really, Unloader and Trains

### ISSUE THREAD

**Issue by hottheadliberty** [Stephan-S/FS19_AutoDrive#1806]
**Question:**   
I've got a question...
Am I right in assuming that the reason my unloader is doing nothing is because my combine on the opposite side of the field is out of the unloader's detection range?

and is there a way to have autodrive stop my vehicles for trains?

**Comment by Axel32019** [comment]
Hello @hottheadliberty ,
Select the first destination to same in harvester and unloader(s), do not activated AD at harvester, set in harvester settings Find Driver to NO.
What you mean with "and is there a way to have autodrive stop my vehicles for trains?" ?
Did you enable collision detection?

**Comment by hottheadliberty** [comment]
Yes collision detection is on but it seems to completely ignore the train gates when the come down

**Comment by Axel32019** [comment]
What happen if you drive manual to / through the closed gate?
